dockerfiles/Dockerfile.apiserver

```diff
@@ -0,0 +1,52 @@
+# syntax=docker/dockerfile:1@sha256:e87caa74dcb7d46cd820352bfea12591f3dba3ddc4285e19c7dcd13359f7cefd
```

Comment:
instead of creating a dockerfiles directory, put each dockerfile in the correct directory according to his suffix
do for all dockerfiles